docs/modules/plugins/pages/plugin-mobile-app.adoc

```diff
@@ -475,4 +475,39 @@ When I change Appium session settings:
 |allowInvisibleElements|true |
 ----
 
+=== Select date in date picker
+
+==== *_Info_*
+
+Selects a next or previous picker wheel value in date picker
+
+[WARNING]
+Supported only for `iOS` platform
+Picker wheel argument must be `XCUIElementTypePickerWheel`
+
+==== *_Wording_*
+
+[source,gherkin]
+----
+When I select $direction value in picker wheel located `$locator`
+----
+
+==== Properties
+
+* `mobile-application.date-picker.offset` - see `offset` argument from http://appium.io/docs/en/writing-running-appium/ios/ios-xctest-mobile-gestures/index.html#mobile-selectpickerwheelvalue[Select Picker Wheel Value], default value is `0.1`
```

Comment:
it's doubtful solution: can't the app have multiple picker wheels with different offsets?

Reply:
usually picker wheel is wrapped into separate component and used across all places

Reply:
but that's not necessary, so it's not an app attribute